### PR TITLE
Fixes splitter_moved

### LINKED ISSUE #140: BUG: TypeError splitter_moved
Since #137 

```
(bluesky_2023_2) [iMac ~/src/gemviz] python gemviz/app.py
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
Traceback (most recent call last):
  File "/Users/fannysimoes/src/gemviz/gemviz/bluesky_runs_catalog.py", line 84, in splitter_moved
    setattr(self, deadline, time.time() + self.motion_wait_time)
TypeError: attribute name must be string, not 'int'
```

### PR DESCRIPTION
**Pull request by rodolakis**
- close #140 

Also removed unused argument (`pos`,`index`) and variable (`deadtime`)

### CODE REVIEW

File: gemviz/bluesky_runs_catalog.py [hunk 1]
```diff
@@ -78,13 +78,12 @@ def catalog(self):
     def catalogName(self):
         return self.parent.catalogName()
 
-    def splitter_moved(self, key, pos, index):
-        deadline = getattr(self, f"{key}_deadline", 0)
+    def splitter_moved(self, key):
```
Comment (prjemian) [inline gemviz/bluesky_runs_catalog.py:81]: I want to check this (later).  

My concern: This is a slot. Isn't `pos, index` emitted by the signal this slot receives?  Even though this slot code ignores them, we should leave them as-is or switch to a `*args` to collect them.

Comment (rodolakis) [reply gemviz/bluesky_runs_catalog.py:81]: I see. Let me know.

Comment (prjemian) [reply gemviz/bluesky_runs_catalog.py:81]: Here's a screen view with `*args, **kwargs` added.  There are values in `args` that correspond to the `pos, index` arguments emitted by the signal.

Comment (prjemian) [reply gemviz/bluesky_runs_catalog.py:81]: image

Comment (prjemian) [reply gemviz/bluesky_runs_catalog.py:81]: Here is a screen view with the code in the PR now:

image

Comment (prjemian) [reply gemviz/bluesky_runs_catalog.py:81]: How come this code did not raise an exception?

Comment (rodolakis) [reply gemviz/bluesky_runs_catalog.py:81]: Interesting! I can put them back.

Comment (rodolakis) [reply gemviz/bluesky_runs_catalog.py]: Done.

File: gemviz/bluesky_runs_catalog.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def catalog(self):
     def catalogName(self):
         return self.parent.catalogName()
 
-    def splitter_moved(self, key):
+    def splitter_moved(self, key,*arg,**kwargs):
```
Comment (prjemian) [inline gemviz/bluesky_runs_catalog.py:81]: please black code style to this line (spaces after the commas)

Comment (rodolakis) [reply gemviz/bluesky_runs_catalog.py]: done

**Review by prjemian — APPROVED** [review]
LGTM

**Pull request merged**